web-common/src/features/canvas/inspector/filters/FiltersMapper.svelte

```diff
@@ -19,8 +21,8 @@
   $: inputParams = component.inputParams().filter;
 
   // Enable this when adding support for dimension filters
```

Comment:
Looks like this comment can be deleted now

Reply:
Removed in Canvas: UI Improvements PR 
https://github.com/rilldata/rill/pull/6501/commits/a48f2440ab05def13ac6c0b7487449b7ef99c72b